lib/buildingsync/makers/workflow_maker.rb

```diff
@@ -431,52 +431,337 @@ def cleanup_larger_files(osw_dir)
       end
     end
 
-    def gather_results(dir, baseline_only = false)
+    def get_result_for_baseline_only(dir)
```

Comment:
it looks like we do only gather results for the baseline scenario now in this function and forgot about the other results all together
let's change this function so it take a parameter in "baseline_only" and use it to either return all results or just the results of the baseline